teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, name, specialization:)
```

Comment:
- [ ] You should aslo pass `parent_permission` to the initialize method and set intial value to `true`. Also pass the parameter `name` and initial value of `"Unknown"`